crypto/chacha/asm/chacha-x86_64.pl

```diff
@@ -761,12 +755,12 @@ sub SSSE3_lane_ROUND {
 	movdqa		$xb2,0xa0-0x100(%rcx)
 	movdqa		$xb3,0xb0-0x100(%rcx)
 
-	pshufd		\$0x00,$xt3,$xt0	# "$xc0"
-	pshufd		\$0x55,$xt3,$xt1	# "$xc1"
+	pshufd		\$0x00,$xt3,$xt0	# "\$xc0"
```

Comment:
The idea behind that commentary is to denote which real variable the target register should be "viewed" as. Or "pretends" to be. That's why the name is in quotes. If it upsets the 'strict' it's probably more appropriate to omit $.